pkg/api/helpers.go

```diff
@@ -304,6 +304,21 @@ func LoadBalancerStatusDeepCopy(lb *LoadBalancerStatus) *LoadBalancerStatus {
 	return c
 }
 
+// ValidateReclaimPolicy ensures proper values are used
+// for persistentVolumeReclaimPolicy in PV spec
+func ValidateReclaimPolicy(policy string) bool {
+	if policy == string(PersistentVolumeReclaimDelete) {
```

Comment:
@pmorie - yeah I missed that, I'm removing that function from helpers altogether as it is not needed, instead I can just use a stringset as you suggested to test the values.  tested and updated screenshot above with results - PTAL